Userland/Libraries/LibWeb/Cookie/ParsedCookie.h

```diff
@@ -16,6 +16,7 @@ namespace Web::Cookie {
 struct ParsedCookie {
     String name;
     String value;
+    String same_site_attribute { "Default"sv };
```

Comment:
This should be an enum with a limited set of options

Reply:
I need to figure out how to pull this off, with my basic c++ knowledge
since I come from typescript world in there I would use 
```typescript
const enum StrictSiteMode {
    DEFAULT = "Default",
    NONE = "None",
    STRICT = "Strict",
    LAX = "Lax"
}
```
In c++ seems like these work a bit different
If I do this 
```c++
enum class SameSiteMode {
    Default,
    None,
    Strict,
    Lax
};
struct ParsedCookie {
    String name;
    String value;
    SameSiteMode same_site_attribute { SameSiteMode::Default };
```
I Need to remap it back to string for cookie storage inspector
Is there something in serenity that you guys use for enum to string mapping?
Or there is just a straightforward way and I am overcomplicating this?

Reply:
This looks good!

>  Is there something in serenity that you guys use for enum to string mapping?
> Or there is just a straightforward way and I am overcomplicating this?

There's no magic to it, just like this: https://github.com/SerenityOS/serenity/blob/3f3f45580ab7266258e97cb3cecf1e24716d61c5/Userland/Libraries/LibWeb/CSS/PreferredColorScheme.cpp#L20-L31

Also applies to the struct in `Cookie/Cookie.h` as well, which I missed earlier. No idea why we have two structs, the enum should probably in `Cookie.h`, not `ParsedCookie.h`

Reply:
tnx for the guidance, I really appreciate it
I pushed the latest changes, hopefully I did it correct